### DISCUSSION TITLE
Getting current trackingAuthorizationStatus?

### DISCUSSION THREAD

**Discussion by pixelone** [admob-plus/admob-plus#364] [Fund Features]
Currently using the Capacitor 3 plugin:   `"@admob-plus/capacitor": "^1.11.0",`

Is it possible to get the current trackingAuthorizationStatus without requestTrackingAuthorization() ?   In some cases may be useful to have a method that returns the current status.

Appreciate all your time on this plugin, its much appreciated.

**Comment by ratson** [comment]
No API for that yet, feel free to send PR.

**Reply by xiaoxiao99** [reply]
+1 for this feature because this would be useful if we want to show some dialog to explain why tracking is needed.
Was also discussed in [another issue](https://github.com/admob-plus/admob-plus/issues/182#issuecomment-683267068)
